bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResumingStreamingResultScanner.java

```diff
@@ -60,6 +62,7 @@
   private long rowCount = 0;
   // The number of times we've retried after a timeout
   private AtomicInteger timeoutRetryCount = new AtomicInteger();
+  private RpcMetrics rpcMetrics;
```

Comment:
There's a fair amount of mutable instance state here so if it can be made final I think it should be.